modules/item/item-wfrp4e.js

```diff
@@ -1583,8 +1583,10 @@ export default class ItemWfrp4e extends Item {
     let skill
     if (this.type == "weapon") {
       skill = skills.find(x => x.name.toLowerCase() == this.skill.value.toLowerCase())
-      if (!skill)
-        skill = skills.find(x => x.name.toLowerCase().includes(`(${this.WeaponGroup.toLowerCase()})`))
+      if (!skill) {
+        let weaponGroupName = game.wfrp4e.config.weaponGroups[this.weaponGroup.value];
+        skill = skills.find(x => x.name.toLowerCase().includes(`(${weaponGroupName.toLowerCase()})`))
+      }
```

Comment:
What's the purpose here? `this.WeaponGroup` is the same as `game.wfrp4e.config.weaponGroups[this.weaponGroup.value];`

Reply:
you've got me here. Probably i was debugging something and didnt notice the lowercase field vs uppercase property getter. I will revert this one.